gubernator/main.py

```diff
@@ -64,6 +65,10 @@ def get_github_client():
         return None
 
 
+def get_app_config():
+    with open('config.yaml') as config_file:
+        return yaml.load(config_file)
```

Comment:
safe_load here and elsewhere? We should not need to execute arbitrary code

http://pyyaml.org/wiki/PyYAMLDocumentation#LoadingYAML

Reply:
It's not arbitrary input, if you can write config.yaml you can write main.py.

Reply:
That is mildly terrifying.